### PR TITLE
Update JitBench dev branch to avoid version mismatches with nuget feeds and the azure sdk feed

### PR DESCRIPTION
**Pull request by davmason**
JitBench broke for a short while because the versions on nuget for the shared runtime and on the azure SDK feed were different. Adding a wildcard in MusicStore.csproj meant it assumed we had the latest available installed, but actually we were two versions behind. We can take advantage of a .net SDK 2 feature and omit the wildcard to fix this, but have to update to the master channel when installing. Also there are some updates the the csproj that are necessary.

@rynowak PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Get the newest dotnet Shared Runtime as 'repo local'
 
 `.\Dotnet-Install.ps1 -SharedRuntime -InstallDir .dotnet -Channel master -Architecture x64`
 
-`.\Dotnet-Install.ps1 -InstallDir .dotnet -Architecture x64`
+`.\Dotnet-Install.ps1 -InstallDir .dotnet -Architecture x64 -Channel master`
```
Comment (rynowak) [inline README.md]: Small nitpick, put the arguments in the same order as the above command

**Comment by davmason** [comment]
@rynowak I'm not sure why the commits that should have already been merged in the last PR are showing up here. The only change should be 19bb0d8d78a699a5b351a0d1cc2fcf38e9715869

**Comment by rynowak** [comment]
OK, changes look good to me. I'll just cherry pick 19bb0d8 and merge it.

**Comment by rynowak** [comment]
Merged 6dd86ec5cf8f8099bde57a864757520b999ec5f8

**Pull request closed without merging**